### PR TITLE
Form UI Fixes

### PR DESCRIPTION
**Pull request by christianmenk**
This is the branch Scott and I had worked on for UI fixes. Some of the changes had to be removed and placed into a new branch. 

**Changes Made**

- radio button questions now fade in
- custom radio buttons for forms
- custom check boxes for forms
- font-awesome plus and minus signs
- moved question in class questions (in intake packet) from placeholder to label above box

Big thanks to @Scott123180 for helping us out with these changes!

### CODE REVIEW

File: views/forms/intake_packet.php [hunk 1]
```diff
@@ -747,222 +747,299 @@
...
+                                        <label class="custom-control custom-radio" for="drug_alcohol_abuse_no">
+                                            <input type="radio" id="drug_alcohol_abuse_no" name="drug_alcohol_abuse" class="custom-control-input" value="No">
+                                            <span class="custom-control-indicator"></span>
+                                            <span class="custom-control-description">No</span>
                                         </label>
                                     </div>
+                                    <!-- End Q: Drug/Alcohol Abuse -->
 
-                                    <div class="form-group hidden-field row drug_alcohol_abuse_div answer_yes">
+                                    <div class="form-group hidden-field row drug_alcohol_abuse_div_yes answer_yes">
```
Comment (blackjack26) [inline views/forms/intake_packet.php:763]: The label on line 764 is "Please explain:" and the placeholder is also "Please explain...". In this case I don't think we need the placeholder.

**Review by sbohner14335 — APPROVED** [review]
Looks good to me.

File: views/forms/intake_packet.php [hunk 2]
```diff
@@ -747,222 +747,299 @@
...
-                                    <div  class="form-group hidden-field row mandated_div answer_yes">
+                                    <div  class="form-group hidden-field row mandated_div_yes answer_yes">
                                         <label class="col-form-label col-sm-2" for="reason_mandated">Mandate Reason:</label>
                                         <div class="col-sm-5">
                                             <input type="text" class="form-control" id="reason_mandated" name="reason_mandated" placeholder="Reason you were mandated (be specific)">
                                         </div>
                                     </div>
 
-                                    <div  class="form-group hidden-field row mandated_div answer_yes">
+                                    <div class="form-group hidden-field row mandated_div_no answer_no">
```
Comment (blackjack26) [inline views/forms/intake_packet.php:913]: Same thing as above, the label and placeholder are kind of redundant.

File: views/forms/intake_packet.php [hunk 3]
```diff
@@ -747,222 +747,299 @@
...
+                                    <!-- Begin Class Takeaway -->
+                                    <div class="row">
+                                        <label class="col-form-label col-sm-12 col-12" for="class_takeaway" style="text-align: left;">What is the most important thing you would like to learn from this class?</label>
+                                    </div>
                                     <div class="form-group row">
-                                        <label class="col-form-label col-sm-2 col-3" for="class_takeaway">Class Takeaway:</label>
-                                        <div class="col-sm-5 col">
-                                            <input type="text" class="form-control" id="class_takeaway" name="class_takeaway" placeholder="What is the most important thing you would like to learn from this class?">
+                                        <div class="col-sm-8 col-8">
+                                            <input type="text" class="form-control" id="class_takeaway" name="class_takeaway" placeholder="Please explain">
```
Comment (blackjack26) [inline views/forms/intake_packet.php]: `placeholder="Please explain..."`

File: views/forms/intake_packet.php [hunk 4]
```diff
@@ -747,222 +747,299 @@
...
+                                    <div class="row">
+                                        <label class="col-form-label col-sm-12 col-12" for="parenting_opinion" style="text-align: left;">What behaviors would keep you from voicing your opinion on your parenting style?</label>
+                                    </div>
                                     <div class="form-group row">
-                                        <label class="col-form-label col-sm-2 col-3" for="parenting_opinion">Parenting:</label>
-                                        <div class="col-sm-5 col">
-                                            <input type="text" class="form-control" id="parenting_opinion" name="parenting_opinion" placeholder="What behaviors would keep you from voicing your opinion on your parenting style?">
+
+                                        <div class="col-sm-8 col-8">
+                                            <input type="text" class="form-control" id="parenting_opinion" name="parenting_opinion" placeholder="Please explain">
```
Comment (blackjack26) [inline views/forms/intake_packet.php]: `placeholder="Please explain..."`

File: views/forms/intake_packet.php [hunk 5]
```diff
@@ -747,222 +747,299 @@
...
+                                    <!-- Begin Class Participation -->
+                                    <div class="row">
+                                        <label class="col-form-label col-sm-12 col-12" for="class_participation" style="text-align: left">What do you need from this class to feel safe and fully participate?</label>
+                                    </div>
                                     <div class="form-group row">
-                                        <label class="col-form-label col-sm-2 col-3" for="class_participation">Class Participation:</label>
-                                        <div class="col-sm-5 col">
-                                            <input type="text" class="form-control" id="class_participation" name="class_participation" placeholder="What do you need from this class to feel safe and fully participate?">
+                                        <div class="col-sm-8 col-8">
+                                            <input type="text" class="form-control" id="class_participation" name="class_participation" placeholder="Please explain">
```
Comment (blackjack26) [inline views/forms/intake_packet.php]: Add the ellipsis to the placeholder: `placeholder="Please explain..."`

**Review by blackjack26 — CHANGES_REQUESTED** [review]

File: views/forms/intake_packet.php (hunk 1 above)
Comment (christianmenk) [reply views/forms/intake_packet.php:763]: Fixed all placeholder issues

**Review by blackjack26 — APPROVED** [review]

**Pull request merged**